bbb-convert-stage-5.sh

```diff
@@ -215,6 +213,13 @@ install_files() {
   sudo ln -fs /sbin/fw_printenv $rootfs_dir/usr/bin/fw_printenv
   sudo ln -fs /sbin/fw_setenv $rootfs_dir/usr/bin/fw_setenv
 
+  #Create links for grub
+  local current_directory=$(pwd)
```

Comment:
I would avoid changing directories here. You have available full paths for mounted boot & rootfs partitions. Just use them...

Reply:
I need to create relative symbolic links. Absolute symbolic links does not work under different root file system.

Reply:
done